Dockerfile.mayhem

```diff
@@ -0,0 +1,29 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
Same thing here, do we have to clone twice? Can we just copy the lib from the build stage?

Reply:
Huh; it actually wasn't even necessary.